tests/pd/mock/server.rs

```diff
@@ -33,12 +33,7 @@ impl Server {
     pub fn run<C>(eps_count: usize, handler: Arc<Service>, case: Option<Arc<C>>) -> Server
         where C: Mocker + Send + Sync + 'static
     {
-        let eps: Vec<SocketAddr> = [0].iter()
-            .cycle()
-            .take(eps_count)
-            .map(|_| ("127.0.0.1", 0).to_socket_addrs().unwrap().next().unwrap())
-            .collect();
-
+        let eps = vec![("127.0.0.1", 0).to_socket_addrs().unwrap().next().unwrap(); eps_count];
```

Comment:
https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html

`&'a [SocketAddrs]` impls `ToSocketAddrs`, not `Vec<(&str, u16)>` or `&'a [(&str, u16)]`.